Enum_mod.thy

```diff
@@ -7,7 +7,7 @@ begin
 
 subsection \<open>Small finite types as arithmetic modulo its cardinal\<close>
 
-text \<open>Similar types are defined as finite_1 up to finite_5 in Enum.thy
+text \<open>Similar types are defined as @{term finite_1} up to @{term finite_5} in Enum.thy
```

Comment:
(Using `@{term}`, not `@{const}` here to avoid an import and still get a teletypish font. No sure if that's the preferred way of going on about it.)